### ISSUE TITLE
Invalid hardware configuration after re connect keyboard

### ISSUE THREAD

**Issue by ert78gb** [UltimateHackingKeyboard/firmware#906]
[uhk-firmware-11.1.0-e6d37495.tar.gz](https://github.com/user-attachments/files/15984091/uhk-firmware-11.1.0-e6d37495.tar.gz)

The attached firmware returns with invalid hardware configuration after reconnect to the computer.
If I update the hardware config with the `./packages/usb/write-hardware-config.ts --log=usb --vid 0x1d50 --pid 0x6125 --usb-interface 2 uhk80v1 iso` then Agent could read the correct user config again, until reconnecting my UHK.

Reproduction
- build the lates agent80/master
- unplug and plug the uhk 80 keyboard
- start the Agent
- Agent will show the `Please power cycle your keyboard (Invalid hardware configuration: Invalid signature)` error
- Write the hardware config `./packages/usb/write-hardware-config.ts --log=usb --vid 0x1d50 --pid 0x6125 --usb-interface 2 uhk80v1 iso`
- start the Agent. It will read the correct hardware config.

the invalid hardware config
```javascript
{
  signature: '\x00\x01\x00\x00\x00�\x10',
  _majorVersion: 0,
  _minorVersion: 0,
  _patchVersion: 6,
  _brandId: 77,
  _deviceId: 121,
  _uniqueId: 1263031584,
  isVendorModeOn: true,
  isIso: false
}
```

The valid hardware config
``` javascript
{
  signature: 'UHK',
  _majorVersion: 1,
  _minorVersion: 0,
  _patchVersion: 0,
  _brandId: 0,
  _deviceId: 3,
  _uniqueId: 1118999665,
  isVendorModeOn: false,
  isIso: false
}
```

The `_uniqueId` can be different because it is the unique identifier of the keyboard.

Agent shows the `Please power cycle your keyboard (Invalid hardware configuration: Invalid signature)` when `signature` field has invalid value. Relevant code https://github.com/UltimateHackingKeyboard/agent80/blob/fd033cd09c8e986505b0300c0aa034781f918bba/packages/uhk-common/src/util/helpers.ts#L15

**Comment by ert78gb** [comment]
- Yesterday, I updated the firmware with  
[uhk-firmware-11.1.0-5aef8a26.tar.gz](https://github.com/user-attachments/files/16041344/uhk-firmware-11.1.0-5aef8a26.tar.gz)
- If I remember well I wrote the hardware config and tested. The keyboard remembered to the hardware config after I unplugged and plugged.

- I started to develop a new Agent feature and when I Agent tried to read the hardware config I got the error. Unfortunately, I forgot to save the response. 
- If I now reconnect the keyboard it remembers to the hardware config.

I store the keyboard unplugged from the computer and removed the battery because it did not turn off the keyboard backlighting. 

Is it possible if the keyboard does not get power for more hours then it forget the config.
I will test it tomorrow again I just wanted to share it
